### PR TITLE
correct SUMOLOGIC_URL referent

### PR DESCRIPTION
**Pull request by jeff-d**
Users need to use the API endpoint, not the service endpoint (e.g. login page).

## Purpose of this pull request

This pull request corrects and clarifies details around which SUMOLOGIC_URL is used with the example curl commands.

## Select the type of change

- [X ] Minor Changes - Typos, formatting, slight revisions

### CODE REVIEW

File: docs/observability/aws/other-configurations-tools/add-new-aws-service.md [hunk 1]
```diff
@@ -72,6 +72,13 @@ Once all the tags are checked and identified in the metrics, we can
 update the existing hierarchy to show the new AWS Service. Follow the
 steps below to update the existing hierarchy :
 
+:::note
+- **ACCESS_ID** and **ACCESS_KEY** - Replace parameters with your sumo logic access Id and access key.
```
Comment (kimsauce) [inline docs/observability/aws/other-configurations-tools/add-new-aws-service.md:76]:
```suggestion
- **ACCESS_ID** and **ACCESS_KEY** - Replace parameters with your Sumo Logic access Id and access key.
```

File: docs/observability/aws/other-configurations-tools/add-new-aws-service.md [hunk 2]
```diff
@@ -72,6 +72,13 @@ Once all the tags are checked and identified in the metrics, we can
...
+- **SUMOLOGIC_URL** - Replace with [API Endpoint URL](https://help.sumologic.com/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) as per deployment.
+- **ID** - Replace with the hierarchy ID as present in the JSON output from Step 1.
+- **JSON_CONTENT_AFTER_UPDATE** - Replace with the JSON updated with new AWS service after Step 2.
```
Comment (kimsauce) [inline docs/observability/aws/other-configurations-tools/add-new-aws-service.md:79]:
```suggestion
- **ACCESS_ID** and **ACCESS_KEY**. Replace parameters with your Sumo Logic access Id and access key.
- **SUMOLOGIC_URL**. Replace with [API Endpoint URL](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) as per deployment.
- **ID**. Replace with the hierarchy ID as present in the JSON output from Step 1.
- **JSON_CONTENT_AFTER_UPDATE**. Replace with the JSON updated with new AWS service after Step 2.
```

**Review by kimsauce — APPROVED** [review]

File: docs/observability/aws/other-configurations-tools/add-new-aws-service.md [hunk 3]
```diff
@@ -72,6 +72,13 @@ Once all the tags are checked and identified in the metrics, we can
 update the existing hierarchy to show the new AWS Service. Follow the
 steps below to update the existing hierarchy :
 
+:::note
+- **ACCESS_ID** and **ACCESS_KEY**. Replace parameters with your Sumo Logic access Id and access key.
+- **SUMOLOGIC_URL**. Replace with [API Endpoint URL](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) as per deployment.
+- **ID**. Replace with the hierarchy ID as present in the JSON output from Step 1.
+- **JSON_CONTENT_AFTER_UPDATE**. Replace with the JSON updated with new AWS service after Step 2.
+:::
```
Comment (jpipkin1) [inline docs/observability/aws/other-configurations-tools/add-new-aws-service.md:80]:
```suggestion
In the code samples in the following steps:
- **ACCESS_ID** and **ACCESS_KEY**. Replace with your Sumo Logic access ID and access key.
- **SUMOLOGIC_URL**. Replace with the [API Endpoint URL](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) for your deployment.
- **ID**. Replace with the hierarchy ID as present in the JSON output from Step 1 below.
- **JSON_CONTENT_AFTER_UPDATE**. Replace with the JSON updated with new AWS service after Step 2 below.
:::
```

**Review by jpipkin1 — APPROVED** [review]
Approved, @jeff-d. I made a few minor updates (see my [commit](https://github.com/SumoLogic/sumologic-documentation/pull/3844/commits/571c35fe273c51af0a8042beea9419c29863e892)).

**Pull request merged**